cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -168,6 +171,7 @@ data PartialNodeConfiguration
        , pncLoggingSwitch  :: !(Last Bool)
        , pncLogMetrics     :: !(Last Bool)
        , pncTraceConfig    :: !(Last PartialTraceOptions)
```

Comment:
The question here is, should be include the socket path and mode configuration within `TraceOptions`/`PartialTraceOptions`?

Reply:
@Jimbo4350, 

- if you mean the `TraceConfig` type, which is structurally carrying the config for new tracing -- it could be, but then it would have to live in the `trace-dispatcher` library, since that `TraceOptions` type is defined there.  I did not think it was a good option, frankly.
- if you indeed mean the `TraceOptions` type, then, well -- it structurally is just a tracing system selector data type.  So not sure socket paths belong in it..

Reply:
It's a difficult question -- but I'm open to discuss!

Reply:
Alright I think let's just rename it to `ncTraceForwardSocket` for now and that should be good.

Reply:
@Jimbo4350, done.

Reply:
I'm happy to hear other suggestions `ncForwardSocket` doesn't mean anything to me. `ncTraceForwardSocket` tells me that this socket is forwarding traces. What do you have in mind?

Reply:
I just saw this: https://github.com/input-output-hk/cardano-node/pull/3933/commits/3f64c4c5597bf8cf767b84652d912b70eedad631#r888707952

`ncForwardingSocket` ?

Reply:
@Jimbo4350, on a second thought that's fine, actually -- my initial intent was to reflect on `cardano-tracer` somehow, but as @denisshevchenko added, that's overly specific.

I was initially slightly dissatisfied by `TraceForward` being too reflective of the legacy `trace-forwarder` backend, but indeed -- we can simply now read that as just "socket for forwarding traces", which makes sense.